browser/src/canvas/sections/URLPopUpSection.ts

```diff
@@ -102,7 +102,11 @@ class URLPopUpSection extends HTMLObjectSection {
 		}
 
 		document.getElementById(this.copyButtonId).onclick = () => {
-			app.map.sendUnoCommand('.uno:CopyHyperlinkLocation', params);
+			// If _navigatorClipboardWrite is available, use it.
+			if (L.Browser.clipboardApiAvailable || window.ThisIsTheiOSApp)
+				app.map._clip.filterExecCopyPaste('.uno:CopyHyperlinkLocation', params);
+			else // Or use previous method.
+				app.map.sendUnoCommand('.uno:CopyHyperlinkLocation', params);
```

Comment:
can't we move that `if` into `sendUnoCommand`?
That will make us sure we will not forget it when we add new call to that uno command.
Also makes less code needed :)

https://github.com/CollaboraOnline/online/blob/8add6a52b5c850e27d4545494e17d8fec617fe4e/browser/src/control/Toolbar.js#L357